junit/mockwebserver/src/main/java/io/fabric8/mockwebserver/dsl/ReturnOrWebsocketable.java

```diff
@@ -15,5 +15,6 @@
  */
 package io.fabric8.mockwebserver.dsl;
 
-public interface ReturnOrWebsocketable<T> extends Returnable<T>, WebSocketable<WebSocketSessionBuilder<T>>, Replyable<T> {
+public interface ReturnOrWebsocketable<T>
+    extends Returnable<T>, WebSocketable<WebSocketSessionBuilder<T>>, Replyable<T>, Failable<T> {
```

Comment:
This was the minimal change to make `io.fabric8.kubernetes.client.http.AbstractHttpPostTest#expectFailure` possible without shutting down the mock webserver.

Reply:
I've killed off the changes to the mockwebserver and replaced them with a dead socket so the initial connection always fails.